src/cgi/wwsympa.fcgi.in

```diff
@@ -10613,7 +10613,7 @@ sub do_edit_list {
     }
 
     my $data_source_updated_member = 1
-        if grep { $config->get_change($_) }
+		if grep { exists $config->{_changes}{$_} }
```

Comment:
What is the motivation of this change?

Reply:
I noticed the previous code did not work. I could not understand why it failed and found this bypass to make it work. Sorry, I should have added a comment to explain his in the PR.